crates/bevy_render/src/texture/image_texture_conversion.rs

```diff
@@ -196,6 +197,18 @@ impl Image {
                 self.data,
             )
             .map(DynamicImage::ImageRgba8),
+            TextureFormat::Bgra8UnormSrgb => ImageBuffer::from_raw(
```

Comment:
Why do we introduce the `TextureFormat::Bgra8UnormSrgb` here? Is there a reason `bgra` might be preferred over `rgba`?

Reply:
Seconding the confusion about why this is part of this PR. I don't mind it, just a bit confused.

Reply:
wgpu apparently quite likes this format for swapchain textures - this was necessary to get the save to disk functionality working on my machine.

Reply:
Interesting. Can you please leave a comment here to that effect then?